src/main/resources/js/config/open-poker-config.js

```diff
@@ -0,0 +1,16 @@
+AJS.$(document).ready(() => {
+	const initialize = () => {
+		let $allowedProjectsSelect = AJS.$("#allowedProjectsSelect");
+		$allowedProjectsSelect.auiSelect2(
+			{
+				placeholder: "Open poker will be available for each project",
+				allowClear:  true
+			});
+
+		$allowedProjectsSelect.on("change", () => {
+			AJS.$("#allowedProjects").val(AJS.$("#allowedProjectsSelect").val());
+		});
+	};
+
+	initialize();
+});
```

Comment:
(nit) could you please add an empty line

Reply:
Sure, any reason why?

Reply:
It's a unix standard - https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline.